llm_on_ray/inference/serve.py

```diff
@@ -89,57 +71,13 @@ def main(argv=None):
         type=str,
         help="Inference configuration file in YAML. If specified, all other arguments will be ignored.",
```

Comment:
need to revise

Reply:
I'm wondering how to revise it, It's correct for --config right?